test-cases/tests/performance/k03-run-performance-test-against-rhsso.md

```diff
@@ -11,31 +11,37 @@ Note: this test should only be performed at a time it will not affect other ongo
 You should have an AWS IAM account in [QE-AWS account](https://068334777414.signin.aws.amazon.com/console)
 
 ## Steps
-
-1. Login into [QE-AWS account](https://068334777414.signin.aws.amazon.com/console) using your AWS IAM account credentials and select the region to **London**.
-2. Create a new **t2.small** ec2 instance from the sso-load-testing AMI. Set the Auto-assign public IP to Enable and select Network as **sso-load-test-vpc** and subnet as **sso-load-test-subnet**. Configure security group to SSH with "myIP".
-3. SSH into the instance using the public IPv4 address and the centos user. `ssh -i <pemfilename>.pem centos@<public IP>`
-4. `cd keycloak` and `git pull`
+1. Get access to ec2 instance to run tests
+   -  Option 1 - Use existing ec2 instance:
+      1. Contact Stephin Thomas for details on accessing this ec2 instance
```

Comment:
@STEPHINRACHEL can we put the credentials for the ec2 instance in the [vault](https://gitlab.cee.redhat.com/integreatly-qe/vault), then we can just put the link to the vault here and we would not need a second option, wdyt?